context/app/api/client.py

```diff
@@ -113,5 +111,39 @@ def get_provenance(self, uuid):
 
         return provenance
 
+    def get_vitessce_conf(self):
+        if self.is_mock:
```

Comment:
Is this a temporary code until the APIs are up?

Reply:
Mostly: 
- Some APIs are "up"... but they are only accessible from white-listed IPs, so I wanted some way to run the app when out of office.
- One place that isn't on the white list is the Travis server. If the requests were client side, we could mock them in Cypress, but the server side is a black box.